### PR TITLE
feat: added Spark4AI and Ppgs

### PR DESCRIPTION
**Pull request by Ananya-m0140**
Issue(#11 )

### CODE REVIEW

File: src/data/societies.ts [hunk 1]
```diff
@@ -751,6 +751,33 @@ export const SOCIETIES_ALL: ISociety[] = [
     links: {
       fb: 'https://www.facebook.com/kgpspeedcubers/',
     }
+  },
+  {
+    symbol: 'Sa',
+    name: "Spark4AI",
```
Comment (harshkhandeparkar) [inline src/data/societies.ts:757]: Is this an open for all society?

File: src/data/societies.ts [hunk 2]
```diff
@@ -751,6 +751,33 @@ export const SOCIETIES_ALL: ISociety[] = [
     links: {
       fb: 'https://www.facebook.com/kgpspeedcubers/',
     }
+  },
+  {
+    symbol: 'Sa',
```
Comment (harshkhandeparkar) [inline src/data/societies.ts]: Symbol `Sa` is already used for TSA.

File: src/data/societies.ts [hunk 3]
```diff
@@ -751,6 +751,33 @@ export const SOCIETIES_ALL: ISociety[] = [
...
+      linkedin: 'https://www.linkedin.com/company/spark4ai/'
+    }
+  },
+  {
+    symbol: 'Pg',
+    name: "Public Policy and Governance Society",
+    year: 2023,
+    size: 'L',
+    tsg_recognized: false,
+    description: "Public Policy and Governance Society, IIT Kharagpur serves as a dynamic platform for impassioned students to rigorously examine, deliberate upon, and actively engage with current public Policy and governance issues.",
```
Comment (harshkhandeparkar) [inline src/data/societies.ts]: A more neutral and descriptive sentence would be better here. This feels very GPT'd.

File: src/data/societies.ts [hunk 4]
```diff
@@ -751,6 +751,33 @@ export const SOCIETIES_ALL: ISociety[] = [
...
+    block: 'W',
```
Comment (harshkhandeparkar) [inline src/data/societies.ts]: I would suggest adding this to the `U` or `O` block instead.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: src/data/societies.ts (hunk 1 above)
Comment (Ananya-m0140) [reply src/data/societies.ts:757]: Yes

File: src/data/societies.ts (hunk 2 above)
Comment (harshkhandeparkar) [reply src/data/societies.ts]: Not resolved.

Comment (Ananya-m0140) [reply src/data/societies.ts]: Renamed to Sk

File: src/data/societies.ts (hunk 4 above)
Comment (Ananya-m0140) [reply src/data/societies.ts]: Added to O block

**Comment by harshkhandeparkar** [comment]
@Ananya-m0140 can you rebase the branch?

**Review by harshkhandeparkar — APPROVED** [review]

**Pull request merged**